### PR TITLE
[A.Zhukevych]Tasks 61-70

### PR DESCRIPTION
**Pull request by Hollister009**

### CODE REVIEW

File: tasks/62_swap-cases/swap-cases.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const swapCases = (swap) => {
```
Comment (tarasbobak) [inline tasks/62_swap-cases/swap-cases.js]: Very good

File: tasks/63_insert-string/insert-string.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export const insert = (str, insertable, position = 0) => {
```
Comment (tarasbobak) [inline tasks/63_insert-string/insert-string.js]: 👍

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export const smallestCommons = (arr) => {
+  const rangeArr = [Math.min(...arr)];
+  const maxNum = Math.max(...arr);
+  let num = rangeArr[0] + 1;
+  while (num <= maxNum) {
+    rangeArr.push(num);
+    num++;
+  }
+  let curN = maxNum * 2;
```
Comment (tarasbobak) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: This code:
```
let curN = maxNum * 2;
  let condition = rangeArr.every((el) => el % curN === 0);
  while (!condition) {
    curN += maxNum;
    condition = rangeArr.every((el) => curN % el === 0);
  }
```

is almost total duplication of itself.
It definitely could be written better

File: tasks/69_unpack-elements/unpack-elements.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+export const unpackArray = (multiArray) => {
+  const resArr = [...multiArray];
+  return resArr.reduce((sum, cur) => {
+    if (!Array.isArray(cur)) return sum.concat(cur);
```
Comment (tarasbobak) [inline tasks/69_unpack-elements/unpack-elements.js]: if else should ALWAYS have curly braces.
if () {
} else {
}

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+export const smallestCommons = (arr) => {
+  const rangeArr = [Math.min(...arr)];
```
Comment (tarasbobak) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: If you know that one is smaller you know that the other one is bigger.
I would sort or do smth similar

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  const maxNum = Math.max(...arr);
+  let num = rangeArr[0] + 1;
+  while (num <= maxNum) {
```
Comment (tarasbobak) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: create a utility function which get start num and end num will create an array of sequence of numbers between them

**Comment by tarasbobak** [comment]
Awesome job overly!
Please rewrite smaller-common-multiple.js

File: tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+export const sumFibs = (num) => {
+  const fiboArr = fibonacci(num);
+  return fiboArr
+      .filter((el) => el % 2 !== 0 && el <= num)
```
[the remark this answers is not in the document]
Comment (Hollister009) [reply tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js]: ESLint wants: 4

File: tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+      .reduce((sum, cur) => sum + cur, 0);
+};
+
+const fibonacci = (num) => {
+  let sequence = [1, 1];
```
[the remark this answers is not in the document]
Comment (Hollister009) [reply tasks/64_sum-odd-fibonacci/sum-odd-fibonacci.js]: Ok

**Pull request merged**